stable/prometheus-cloudwatch-exporter/templates/deployment.yaml

```diff
@@ -106,6 +109,13 @@ spec:
       securityContext:
 {{ toYaml . | indent 8 }}
     {{- end }}
+      {{ if .Values.serviceAccount.create }}
```

Comment:
`if .Values.serviceAccount.create` should be placed inside the `serviceaccount.yaml` and control whether the service account should be created or not.